website/tailwind.config.ts

```diff
@@ -1,6 +1,18 @@
+import type { Config } from 'tailwindcss';
 import guildConfig from '@theguild/tailwind-config';
 
-export default {
+const config: Config = {
   ...guildConfig,
   content: [...guildConfig.content, 'theme.config.tsx'],
+  theme: {
+    ...guildConfig.theme,
+    extend: {
+      colors: {
+        ...guildConfig.theme.extend.colors,
+        primary: guildConfig.theme.extend.colors['hive-yellow'],
```

Comment:
This doesn't seem to break anything on the Mesh landing page and examples, so we can probably merge the section to have it on master already.